helm/cluster-api/files/bootstrap/bases/kubeadmconfigs.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -12,8 +12,9 @@ metadata:
     app.kubernetes.io/name: cluster-api
     app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
     cluster.x-k8s.io/provider: bootstrap-kubeadm
-    cluster.x-k8s.io/v1alpha2: v1alpha2
```

Comment:
For some reason we don't have this `cluster.x-k8s.io/v1alpha2: v1alpha2` label in the CRD from our fork, even though we ported back `v1alpha2`, it looks like we missed some config. @AverageMarcus do you know where this could be coming from?

Reply:
I'm not sure. Might be we missed the labels? I'm not sure what these are used for (if anything)

Reply:
[If I understand the docs](https://release-0-4.cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#apply-the-contract-version-label-clusterx-k8sioversion-version1_version2_version3-to-your-crds), they say is required "starting with v1alpha3". It seems it's a map to specify which API Version of a provider's CRDs are used for a given CAPI version.